test/katex-spec.js

```diff
@@ -1382,6 +1382,15 @@ describe("An HTML font tree-builder", function() {
         span = "<span class=\"mord mathbb\" style=\"color:blue;\">R</span>";
         expect(markup).toContain(span);
     });
+
+    it("should throw TypeError when the expression is of the wrong type", function() {
+        expect(function() {
+            katex.renderToString({badInputType: "yes"});
+        }).toThrowError(TypeError);
+        expect(function() {
+            katex.renderToString(["foo", {badInputType: "yes"}]);
+        }).toThrowError(TypeError);
+    });
```

Comment:
I added a test case simply for not throwing on correct types being supplied in https://github.com/Khan/KaTeX/pull/540/commits/3dc527683003d918f487e09f4d7dc810e0e5a894 I also think it's good to not advertise using `String` objects. I could see reusing some of the tests from `describe("An HTML font tree-builder", ` with `String` objects if the coverage tests are desirable. I can add those test cases if that is desired.